docs/src/components/LoadingIndicator/Basic.tsx

```diff
@@ -0,0 +1,39 @@
+import {LoadingIndicator, ThemeProvider, ThemeType, useTheme} from 'dooboo-ui';
+
+import type {FC} from 'react';
+import {View} from 'react-native';
+
+export const Component: FC = () => {
+  const {theme} = useTheme();
+
+  return (
+    <View>
+      <View
+        style={{
+          backgroundColor: theme.background,
+          alignSelf: 'stretch',
+          justifyContent: 'center',
+          padding: 30,
+        }}>
```

Comment:
You can try `Styled component` for this. Because this is repeating.
See [this](https://github.com/dooboolab/dooboo-ui/blob/master/docs/src/components/Button/Basic.tsx) as example.